src/library/trackcollection.cpp

```diff
@@ -510,4 +516,68 @@ StorageUnicodeFunctions::likeFunction(sqlite3_context *p,
                                     itrString, endString, E));
 }
 */
+//Private function
+
+bool TrackCollection::settingsPathStatus(){
+    struct stat sb; //this struct will hold the access info of the dir using struct here as below it has function of same name
+    stat(m_pConfig->getSettingsPath().toStdString().c_str(), &sb);  //give the status, m_pConfig->getSettingsPath() will give the
```

Comment:
OK, i try, but I need to convert the QString to c style string as stat is the function of the C library, I checked on net on the stackoverflow, they tell is a good way at http://stackoverflow.com/questions/2523765/qstring-to-char-conversion the third answer